modules/gnome/nixos.nix

```diff
@@ -13,7 +13,7 @@ in {
       description = lib.mdDoc "Whether to style GNOME";
       type = lib.types.bool;
       default = config.stylix.autoEnable 
-             && config.services.xserver.desktopManager.gnome.enable;
+             && (config.services.xserver.desktopManager.gnome.enable or false);
```

Comment:
Does Nix not implement the operators such that `A && (B or false)` early exits similar to `A && B` if `B` is `false`?

Reply:
See linked issue, apparently `config` might not be populated at the time of evaluating this and adding the `or` fixes this.